synapse/handlers/federation.py

```diff
@@ -92,6 +96,24 @@ def get_domains_from_state(state: StateMap[EventBase]) -> List[Tuple[str, int]]:
     return sorted(joined_domains.items(), key=lambda d: d[1])
 
 
+class _BackfillPointType(Enum):
+    # a regular backwards extremity (ie, an event which we don't yet have, but which
+    # is referred to by other events in the DAG)
+    BACKWARDS_EXTREMITY = enum.auto()
```

Comment:
I think  "backward extremity" (not plural) is the correct spelling using the database as the source of truth (and the docs https://github.com/matrix-org/synapse/blob/develop/docs/development/room-dag-concepts.md#backward-extremity but I derived those from the code).